src/code/__osMalloc.c

```diff
@@ -422,15 +422,15 @@ void __osFree_NoLock(Arena* arena, void* ptr) {
     node = (ArenaNode*)((u32)ptr - sizeof(ArenaNode));
     if (node == NULL || node->magic != NODE_MAGIC) {
         osSyncPrintf(VT_COL(RED, WHITE) "__osFree:不正解放(%08x)\n" VT_RST,
-                     ptr); // __osFree: Unauthorized release (%08x)
+                     ptr); // "__osFree: Unauthorized release (%08x)"
```

Comment:
Looks like some of these comments could use another pass to put them above the print if they don't fit in the same line